File: app/src/main/java/com/overswayit/plesnisavezsrbije/events/ViewMesssages.java
package com.overswayit.plesnisavezsrbije.events;

/**
 * Created by lazarristic on 20/02/2019.
 * Copyright (c) 2019 PlesniSavezSrbije. All rights reserved.
 */
public class ViewMesssages {
    public static class ClubContactPhoneCall extends Event<String> { protected ClubContactPhoneCall() {} }
    public static ClubContactPhoneCall clubContactPhoneCall = new ClubContactPhoneCall();

    public static class ClubContactPhoneMessage extends Event<String> { protected ClubContactPhoneMessage() {} }
    public static ClubContactPhoneMessage clubContactPhoneMessage = new ClubContactPhoneMessage();

    public static class ClubContactEmail extends Event<String> { protected ClubContactEmail() {} }
    public static ClubContactEmail clubContactEmail = new ClubContactEmail();
}

File: app/src/main/java/com/overswayit/plesnisavezsrbije/events/ViewMesssages.kt
package com.overswayit.plesnisavezsrbije.events

/**
 * Created by lazarristic on 20/02/2019.
 * Copyright (c) 2019 PlesniSavezSrbije. All rights reserved.
 */
object ViewMesssages {
    var clubContactPhoneCall = ClubContactPhoneCall()
    class ClubContactPhoneCall : Event<String>()

    var clubContactPhoneMessage = ClubContactPhoneMessage()
    class ClubContactPhoneMessage : Event<String>()

    var clubContactEmail = ClubContactEmail()
    class ClubContactEmail : Event<String>()
}
